commands/station.js

```diff
@@ -3,96 +3,79 @@
 const { join } = require('node:path')
 const saturnNode = require('../lib/saturn-node')
 const zinniaRuntime = require('../lib/zinnia')
-const { formatActivityObject } = require('../lib/activity')
-const lockfile = require('proper-lockfile')
-const { maybeCreateFile } = require('../lib/util')
+const { formatActivityObject, activities } = require('../lib/activity')
 const { startPingLoop } = require('../lib/telemetry')
 const bacalhau = require('../lib/bacalhau')
+const fs = require('node:fs/promises')
+const { metrics } = require('../lib/metrics')
+const { paths } = require('../lib/paths')
 
 const { FIL_WALLET_ADDRESS, MAX_DISK_SPACE } = process.env
 
-const station = async ({ core, json, experimental }) => {
+const moduleNames = [
+  'zinnia',
+  'saturn-L2-node',
+  'bacalhau'
+]
```

Comment:
I am surprised that you need to define a list of all valid modules. How comes we did not need this list before?

Reply:
we had it, only in `index.js`